Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -647,6 +647,15 @@ bool CtapAuthenticator::isUVSetup() const
     return m_info.options().clientPinAvailability() == AuthenticatorSupportedOptions::ClientPinAvailability::kSupportedAndPinSet || m_info.options().userVerificationAvailability() == AuthenticatorSupportedOptions::UserVerificationAvailability::kSupportedAndConfigured;
 }
 
+fido::PINUVAuthProtocol CtapAuthenticator::selectPinProtocol() const
+{
+    if (auto& protocols = m_info.pinProtocol()) {
+        if (std::find(protocols->begin(), protocols->end(), 2) != protocols->end())
+            return fido::PINUVAuthProtocol::kPinProtocol2;
```

Comment:
It seems like `m_info.pinProtocol()` should be backed by a `std::set<fido::PINUVAuthProtocol>`!